projects/httpcomponents-client/MimeFieldToStringFuzzer.java

```diff
@@ -0,0 +1,17 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+
+import org.apache.hc.client5.http.entity.mime.MimeField;
+
+public class MimeFieldToStringFuzzer {
+    private static final int nameLength = 300;
+
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
+        // Create objects from fuzzer input
+        final String name = data.consumeString(nameLength);
+        final String value = data.consumeRemainingAsString();
+
+        // fuzzing begins here
+        final MimeField mimeField = new MimeField(name, value);
+        mimeField.toString();
```

Comment:
Unfortunately this class `MimeField` doesn't do much and therefore there is not much to fuzz. I wouldn't onboard this fuzz target, sorry!

Reply:
I have to admit, I was unable to find many great fuzz targets candidates in `httpmime` because of the limited scope of the project, so I might have created useless stuff.
I will remove the `HeaderAddFieldFuzzer`, `HeaderRemoveFieldsFuzzer` and this one.